docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -175,6 +174,29 @@ TBD.
 tests directory
 ----------------
 
+Executing sanity, integration, and unit tests is largely identical to performing these tests within Ansible's mainline code. 
```

Comment:
There is already a whole section in the document on running tests in collections: https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#testing-collections

I think this should be integrated with that section, instead of repeating the information at another place. (Two paragraphs after your new addition, there's already a link to that section.)